### ISSUE TITLE
[Q]: Confusion matrix as in documentation

### ISSUE THREAD

**Issue by yermandy** [wandb/wandb#9228]
### Ask your question

How do we get [this type](https://docs.wandb.ai/guides/integrations/scikit/#confusion-matrix) of the confusion matrix?

<img width="652" alt="Image" src="https://github.com/user-attachments/assets/374d6699-fd96-4269-b733-8a5e76f022ed" />

---

`wandb.sklearn.plot_confusion_matrix(y_true, y_pred, labels)` does something different:

<img width="548" alt="Image" src="https://github.com/user-attachments/assets/b5bbfe1b-77ef-44c7-8ac7-4e87c1cf6bb6" />

**Comment by aajais** [comment]
Hi @yermandy, Thanks for reaching out! There are a few ways to log a confusion matrix to W&B. It looks like you’re currently using our sklearn integration, which first plots the chart and then uploads it as media to W&B. Alternatively, you can create a custom chart and use the summary table to visualize the confusion matrix in a different way.

To create a custom confusion matrix, as outlined in our docs, you’ll need to edit the Vega spec for that chart. The Vega documentation is quite detailed and can help guide you through the process—you can find it here: [Vega Docs](https://vega.github.io/vega/).

I’ve also attached a sample Vega script that you can use as a starting point. By editing the Vega spec provided, you should be able to create the desired visualization.

Let me know if this helps or if you have any further questions

https://github.com/user-attachments/assets/befdbcc4-c8ee-4bf3-b3e2-041598c6a64b

[vega_spec.json](https://github.com/user-attachments/files/18381299/vega_spec.json)

**Comment by yermandy** [comment]
@aajais thank you very much!
